src/views/dialogs/DepositDialog2/DepositForm.tsx

```diff
@@ -96,12 +99,49 @@ export const DepositForm = ({
   });
 
   const [depositSteps, setDepositSteps] = useState<DepositStep[]>();
+  const [currentStep, setCurrentStep] = useState(0);
+  const [showRetryCurrentStep, setShowRetryCurrentStep] = useState(false);
   const [awaitingWalletAction, setAwaitingWalletAction] = useState(false);
 
+  // Helpers for fetching updated values within the useEffect for autoPromptStep
```

Comment:
all of this skipclient/wallet client ref stuff is to unblock myself until the skip team ships an update to their client SDK that i requested 🙏 i am hoping to delete these soon